tensorflow/contrib/lite/python/convert.py

```diff
@@ -131,8 +131,8 @@ def toco_convert_protos(model_flags_str, toco_flags_str, input_data_str):
       stuff = fp_output.read()
       return stuff
     else:
-      raise RuntimeError("TOCO failed see console for info.\n%s\n%s\n" %
-                         (stdout, stderr))
+      raise RuntimeError("TOCO failed. See console for info.\n%s\n%s\n" %
```

Comment:
Could you add a unit test for this?

Reply:
Will do! Give me a bit of time to work on it :+1:

Reply:
@rmlarsen Done!